.github/workflows/certora_recovery.yml

```diff
@@ -34,9 +34,9 @@ jobs:
 
       - name: Install solc
         run: |
-          wget https://github.com/ethereum/solidity/releases/download/v0.8.23/solc-static-linux
+          wget https://github.com/ethereum/solidity/releases/download/v0.8.20/solc-static-linux
           chmod +x solc-static-linux
-          sudo mv solc-static-linux /usr/local/bin/solc-0.8.23
+          sudo mv solc-static-linux /usr/local/bin/solc-0.8.20
```

Comment:
Interesting - why did we lower the compiler version? No issues against it, but we should document it in the PR description.

Reply:
The contract uses 0.8.20 (thanks to @mmv08 for figuring that out), so we updated this.